### ISSUE TITLE
To document/fix: calculations involving `EpicsMotor` `egu` property can be slow

### ISSUE THREAD

**Issue by klauer** [pcdshub/pcdsdevices#550]
We use this at least in `DelayBase` for unit conversion.

Upstream should fix this at some point, but we should be aware of it:
https://github.com/bluesky/ophyd/blob/9a69acd5c016127855d953ab7148ec27db449f8f/ophyd/epics_motor.py#L114-L117

Without auto-monitor, this results in a `.get()`-call with a network hit each time:

```
In [1]: %timeit delay_stage.forward(0)
339 ms ± 7.54 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

**Comment by ZLLentz** [comment]
Right, we should probably ensure that every property access is "fast", that is, if it's important enough to be a property it should be monitored/cached in some way.

**Comment by klauer** [comment]
Largely resolved by #558. Closing for now.
